File: app/src/main/java/com/riders/thelab/data/local/LabDatabase.java
package com.riders.thelab.data.local;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.riders.thelab.data.local.dao.ContactDao;
import com.riders.thelab.data.local.dao.WeatherDao;
import com.riders.thelab.data.local.model.Contact;
import com.riders.thelab.data.local.model.weather.City;

@Database(
        entities = {Contact.class, City.class},
        version = 3,
        exportSchema = false)
public abstract class LabDatabase extends RoomDatabase {

    public static final String DATABASE_NAME = "lab";

    public abstract ContactDao getContactDao();
    public abstract WeatherDao getWeatherDao();
}

File: app/src/main/java/com/riders/thelab/data/local/LabDatabase.kt
package com.riders.thelab.data.local

import androidx.room.Database
import androidx.room.RoomDatabase
import com.riders.thelab.data.local.dao.ContactDao
import com.riders.thelab.data.local.dao.WeatherDao
import com.riders.thelab.data.local.model.Contact
import com.riders.thelab.data.local.model.weather.WeatherData

@Database(
    entities = [Contact::class, CityModel::class, WeatherData::class],
    version = 1,
    exportSchema = false
)
abstract class LabDatabase : RoomDatabase() {
    companion object {
        const val DATABASE_NAME = "lab"
    }

    abstract fun getContactDao(): ContactDao

    abstract fun getWeatherDao(): WeatherDao
}